### PR TITLE
Analysis options, fixes, and formatting

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: shrine/lib/category_menu_page.dart [hunk 1]
```diff
@@ -75,9 +75,8 @@ class CategoryMenuPage extends StatelessWidget {
         padding: EdgeInsets.only(top: 40.0),
         color: kShrinePink100,
         child: ListView(
-            children: _categories
-                .map((Category c) => _buildCategory(c, context))
-                .toList()),
+            children:
+                _categories.map((c) => _buildCategory(c, context)).toList()),
```
Comment (RedBrogdon) [inline shrine/lib/category_menu_page.dart]: nit: throw a penultimate comma in here for the formatter: `...toList(),),`

File: shrine/lib/supplemental/product_columns.dart [hunk 2]
```diff
@@ -27,8 +27,7 @@ class TwoProductCardColumn extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return LayoutBuilder(
-        builder: (BuildContext context, BoxConstraints constraints) {
+    return LayoutBuilder(builder: (context, constraints) {
```
Comment (RedBrogdon) [inline shrine/lib/supplemental/product_columns.dart]: optional nit: I would throw a comma between the brace and parenthesis on line 63 to bump the builder parameter down a line and maintain the one-closing-bracket-per-line indent format.

File: shrine/lib/category_menu_page.dart (hunk 1 above)
Comment (domesticmouse) [reply shrine/lib/category_menu_page.dart]: done

File: shrine/lib/supplemental/product_columns.dart (hunk 2 above)
Comment (domesticmouse) [reply shrine/lib/supplemental/product_columns.dart]: Amusingly, that minor change caused a full relayout of pretty much everything.

**Pull request merged**